file-formats/uiad/type/zif_aff_uiad_v1.intf.abap

```diff
@@ -255,8 +255,8 @@ INTERFACE zif_aff_uiad_v1
       "! Filter value
       filter_value        TYPE string,
 
-      "! <p class="shorttext">Filter Value Type</p>
-      "! Filter value type
+      "! <p class="shorttext">Filter Type</p>
+      "! Filter type
       filter_value_type   TYPE ty_filter_value_type,
```

Comment:
I am not sure whether it is a difference between "filter type" and "filter value type". I know for BAdIs there is a difference between "filter" and "filter value" ;)

Reply:
We would rename field from filter_value_type to filter_type